src/common/offline_transformations/include/mask_attribute.hpp

```diff
@@ -196,8 +201,20 @@ class Mask : public std::vector<std::set<uint64_t>>,
         m_need_initialization = true;
     }
 
+    bool adjust_value() const {
+        return m_adjust_value;
+    }
+
 private:
     bool m_is_shape_like{false};
+    // Flag - if a constant this mask is related to should be pruned
```

Comment:
`Flag - if a constant this mask is related to should be pruned by value` - please change it.

Reply:
Done